### PR TITLE
Update position-elements-with-position-sticky.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Remove incorrect and irrelevant question

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/styling/positioning-tips/position-elements-with-position-sticky.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/web/styling/positioning-tips/position-elements-with-position-sticky.md?ref=b950c48463f10ee0f603919b895268c57ae6666e)

**Comment by kapnobatai137** [comment]
@Stefan-Stojanovic can we replace it with a different question? something like "How can you make the X class sticky?"

**Comment by nem035** [comment]
Yeah, I'd lean on the side of never removing questions. If anything, we should always strive to add new ones (and in this case we should probably add a better one)

File: web/styling/positioning-tips/position-elements-with-position-sticky.md [hunk 1]
```diff
@@ -36,18 +36,22 @@ By simply adding `position: sticky` (vendor prefixed), we can tell an element to
 }
 ```
 
-
 ---
-
 ## Revision
 
-Which of the following browsers *does not* support `position: sticky`? ???
+Can you make the `.some-class` class sticky?
```
Comment (kapnobatai137) [inline web/styling/positioning-tips/position-elements-with-position-sticky.md:42]:
```suggestion
Can you make the `.enki` class sticky?
```

File: web/styling/positioning-tips/position-elements-with-position-sticky.md [hunk 2]
```diff
@@ -36,18 +36,22 @@ By simply adding `position: sticky` (vendor prefixed), we can tell an element to
...
 
-- Chrome
-- Firefox
-- Safari
-- iOS Safari
+```ccss
+.some-class{
```
Comment (kapnobatai137) [inline web/styling/positioning-tips/position-elements-with-position-sticky.md:45]:
```suggestion
.enki {
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
let's change the class name to `enki` instead of `some-class`?

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**